src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/SentenceIdentificationService.kt

```diff
@@ -345,7 +345,7 @@ class SentenceIdentificationService(val hdcedCalculator: HdcedCalculator, val tu
             }
           }
         } else {
-          if (overEighteen && durationGreaterThanSevenYears && sentence.offence.isScheduleFifteenMaximumLife) {
+          if (sentence.sentencedAt.isAfterOrEqualTo(SDS_PLUS_COMMENCEMENT_DATE) && overEighteen && durationGreaterThanSevenYears && sentence.offence.isScheduleFifteenMaximumLife) {
```

Comment:
This should probably be inside the other if block?

Reply:
Done.